ecommerce/extensions/checkout/views.py

```diff
@@ -194,8 +194,21 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
             'has_enrollment_code_product': has_enrollment_code_product,
             'disable_back_button': self.request.GET.get('disable_back_button', 0),
         })
+        context.update({
+            'ar_name': self.get_extended_field(self.request.user.account_details(self.request), 'arabic_name')
+        })
         return context
 
+    def get_extended_field(self, account_details, field_name, default_value=None):
+        """Helper function to extract data from the extended profile."""
```

Comment:
@alfredchavez, use the multi-line docstring format recommended in our coding standards.

```suggestion
        """
        Helper function to extract data from the extended profile.
        """
```